pulpcore/cli/core/user.py

```diff
@@ -2,18 +2,56 @@
 
 import click
 
-from pulpcore.cli.common.context import (
+from pulpcore.cli.common.context import (  # PulpEntityContext,; pass_entity_context,
+    PluginRequirement,
     PulpContext,
-    PulpEntityContext,
     pass_entity_context,
     pass_pulp_context,
 )
-from pulpcore.cli.common.generic import list_command
-from pulpcore.cli.core.context import PulpUserContext
+from pulpcore.cli.common.generic import (
+    create_command,
+    destroy_command,
+    href_option,
+    list_command,
+    lookup_callback,
+    null_callback,
+    pulp_option,
+    show_command,
+    update_command,
+)
+from pulpcore.cli.core.context import PulpUserContext, PulpUserRoleContext
 
 _ = gettext.gettext
 
 
+username_option = pulp_option(
+    "--username",
+    help=_("Username of the {entity}"),
+    expose_value=False,
+    callback=lookup_callback("username", PulpUserContext),
+)
+lookup_options = [
+    href_option,
+    username_option,
+]
+update_options = [
+    click.option(
+        "--password",
+        help=_(
+            "Password for the user. Provide an empty string to disable password authentication."
+        ),
+    ),
```

Comment:
Should we look into the password prompts for this field: https://click.palletsprojects.com/en/8.0.x/options/#password-prompts

Reply:
I guess, we should.

A change my password command would also be nice.

Reply:
Would you mind diverting that to a followup PR?